src/Test/Perf/internalBootstrap.csx

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#load ".\util\test_util.csx"
+#load ".\util\Download_util.csx"
+
+InitUtilities();
+
+// Download the tools(CPC, ViBenchToJson) so that users can use CPC while trying to run the tests on their machines
+DownloadTools();
```

Comment:
Sorry, I meant actually translating the code in DownloadTools to .bat (command shell language).  Shell'ing out to robocopy isn't going to be portable anyway, so having this wrapped in .csx seems like unnecessary indirection (since there's nothing else happening in this script that is easier/simpler to do in C#).